apps/web/src/app/challenge/_components/comments/getCommentRouteData.ts

```diff
@@ -28,6 +29,17 @@ export async function getPaginatedComments({
     },
   });
 
+  const totalReplies = await prisma.comment.count({
+    where: {
+      rootType,
+      parentId: {
+        not: null,
+      },
+      visible: true,
+      ...(rootType === 'CHALLENGE' ? { rootChallengeId: rootId } : { rootSolutionId: rootId }),
+    },
+  });
+
```

Comment:
wish we could do this and totalComments in one query.... anyway this looks good!

Reply:
Agreed! If we ever find a way I'd be glad to refactor 👍🏻